pkgs/data/fonts/monaspace/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "monaspace";
+  version = "1.000";
+
+  src = fetchzip {
+    url = "https://github.com/githubnext/monaspace/releases/download/v${version}/monaspace-v${version}.zip";
+    hash = "sha256-H8NOS+pVkrY9DofuJhPR2OlzkF4fMdmP2zfDBfrk83A=";
+    stripRoot = false;
+  };
+
+  outputs = [ "out" "otf" "variable" ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/fonts/monaspace
+    mkdir -p $out/share/fonts/monaspace-variable
+    mkdir -p $otf/share/fonts/monaspace
+    mkdir -p $variable/share/fonts/monaspace-variable
+
+    cp monaspace-v${version}/fonts/otf/*.otf $out/share/fonts/monaspace
+    mv monaspace-v${version}/fonts/otf/*.otf $otf/share/fonts/monaspace
+    cp monaspace-v${version}/fonts/variable/*.ttf $out/share/fonts/monaspace-variable
+    mv monaspace-v${version}/fonts/variable/*.ttf $variable/share/fonts/monaspace-variable
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Monaspace";
+    longDescription = "An innovative superfamily of fonts for code.";
+    homepage = "https://monaspace.githubnext.com/";
+    license = licenses.ofl;
+    platforms = platforms.all;
+    maintainers = [];
```

If you are not interested in keeping this package, I  can adopt it.

```suggestion
    maintainers = with  maintainers; [ AndersonTorres ];
```

---

And done, thanks for taking ownership.